src/Microsoft.Sbom.Extensions/ISbomConfigProvider.cs

```diff
@@ -49,6 +49,13 @@ public interface ISbomConfigProvider : IDisposable, IAsyncDisposable, IInternalM
     /// <returns></returns>
     IAsyncDisposable StartJsonSerializationAsync();
 
+    /// <summary>
```

Comment:
Is the comment in line 12 correct? (not in this PR, but I noticed it when loading the project in VS)